src/main/java/com/alkemy/ong/entity/NewsEntity.java

```diff
@@ -17,8 +17,8 @@
 @Setter
 @Getter
 @RequiredArgsConstructor
-@SQLDelete(sql = "UPDATE news SET softDeleted = true WHERE id=?")
-@Where(clause = "softDeleted=false")
+@SQLDelete(sql = "UPDATE news SET soft_delete = true WHERE id=?")
+@Where(clause = "soft_delete=false")
```

Comment:
Ana, la clausula y el Set "soft_delete" no deberia ser en CamelCase?, como buena practica solo se usa guion bajo en el nombramiento de las columnas de la base de datos cuando es mas de una palabra.

Reply:
If you dont put it the way Ana did, will throw an error when you try to get an Entity. At least it happend to me.

Reply:
I agree with Pablo I think we should change it in all the entities